client/state/action-log/README.md

```diff
@@ -1,39 +1,73 @@
-# Action Logger
+Action Log
+==========
+
 **A Redux [store enhancer](https://github.com/reactjs/redux/blob/master/docs/Glossary.md#store-enhancer)**
 
-The Action Logger is a Redux store enhancer which records the most recently dispatched redux action
-so they can be inspected in the development environment for debugging and interactive design.
+The Action Log is a Redux store enhancer which records the most recently dispatched Redux action so
+they can be inspected in the development environment for debugging and interactive design.
 
-## Functionality
+### Purpose
 
-### Action logging
+Like the [Redux DevTools](https://github.com/zalmoxisus/redux-devtools-extension), the action log
+is useful to track and inspect actions flowing through the store.
 
-Like the [Redux DevTools](https://github.com/zalmoxisus/redux-devtools-extension), the action
-logger can track a history of the actions flowing through the store.  This can be toggled and the
-size of the history log can be adjusted.
+### Methods
 
-The following examples represent commands and code actually run inside the browser console.
-These are not snippets of bundled JavaScript from Calypso.
+* `actionLog.start()`
+  Enable action recording to history. It is enabled by default.
 
-```js
-// start recording actions
-// enabled by default in the development environment
-actionLog.start()
+* `actionLog.stop()`
+  Disable action recording to history.
+
+* `actionLog.setSize( size )`
+  Set the desired action history size. The history will be flushed periodically to prevent unbounded
+  growth.
+
+* `actionLog.history`
+  Return the current history. The history is an array of action objects.
+
+* `actionLog.filter( query )`
+  Return a filtered history of actions matching the query. See [Query](#query).
+
+* `actionLog.watch( query )`
+  Watch flow of matching actions real-time in the console. Only the single, most recent watch will
+  have effect. See [Query](#query).
+
+* `actionLog.unwatch()`
+  Disable action watching.
+
+### Query
 
-// stop recording actions
-actionLog.stop()
+The `actionLog.filter( query )` and `actionLog.watch( query )` accept a `query`. The following types
+are valid queries which are useful for different situations:
 
-// set how many actions to track (defaults to 100)
-// once enough actions have been recorded in
-// the history log, old actions will be flushed
-// to make room for the new ones
-actionLog.setSize( 1000 )
+* `string`: `actionLog.filter( "COMMENTS_LIKE" )` - Match actions whose type exactly matches the
+  string.
 
+* `RexExp`: `actionLog.filter( /^comments_/i )` - Match actions whose type is matched by the RegExp.
+
+* `PredicateFunction :: action -> bool`:
+  `actionLog.filter( action => action.siteId === interestingSiteId)` - A predicate function that
+  accepts an action and returns `true` if the action matches.
```

Comment:
> and returns `true` if the action matches

Did you mean to say it the other way around - something like "and returns the actions for which the function returns `true`?"

Reply:
🤔 this is an accurate description of the predicate function.

I see now that this may not align with the string and RegExp descriptions, I'll see if I can improve that…

Reply:
Actually, I think these descriptions are pretty decent. I'm going to ship this.